src/Kunstmaan/AdminBundle/Twig/MultiDomainAdminTwigExtension.php

```diff
@@ -78,15 +79,20 @@ public function getSwitchedHost()
     }
 
     /**
-     * @return string
+     * @return bool
      */
     public function switchedHostIsCurrent()
     {
-        return $this->domainConfiguration->getHost() == $this->getSwitchedHost()['host'];
+        return $this->domainConfiguration->getHost() === $this->getSwitchedHost();
```

Comment:
Yes it does return an array when you are using the Multidomain feature. So this will break.

Reply:
Which class returns an array? The multi domain configuration class returns `$this->hosts[$host]`, which is a string, and the normal DomainConfiguration returns null.
I'll test this on one of our multi domain sites to make sure the behaviour is the same though!

Reply:
Ok having checked on a multi domain site it does indeed return an array, so I'll fix the docblock, as it is obviously what was wrong, put the array part back in , and we should merge this anyway, since code has also been tidied up!